### PR TITLE
[release/5.0] Work around nginx installation failure

### PR DESCRIPTION
**Pull request by wtgodbe**
Manually uninstall/delete openssl until https://github.com/dotnet/core-eng/issues/11636 is resolved

### CODE REVIEW

**Comment by BrennanConroy** [comment]
Have we considered the workaround instead?
https://github.com/actions/virtual-environments/issues/1811#issuecomment-732239051

**Comment by wtgodbe** [comment]
Oh nice, I hadn't seen that - giving it a try: https://github.com/dotnet/aspnetcore/pull/28524/commits/44b8a67c053c91582339c23cef3761371911d64f

**Comment by wtgodbe** [comment]
That worked! Thanks @BrennanConroy

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -664,6 +664,12 @@ stages:
         isTestingJob: true
         buildArgs: --all --test "/p:RunTemplateTests=false /p:SkipHelixReadyTests=true" $(_InternalRuntimeDownloadArgs)
         beforeBuild:
+        - bash: "brew uninstall openssl@1.0.2t"
```
Comment (MattGal) [inline .azure/pipelines/ci.yml:667]: this really should strive to be conditional, or ignored;  it will fail once openssl@1.0.2t is not present.

**Review by MattGal — APPROVED** [review]
Looks like it'll work.

**Review by BrennanConroy — APPROVED** [review]

**Comment by BrennanConroy** [comment]
From latest email
>You should be able to use brew as normal again in Hosted MacOS build machines;

Think we can close this.

**Comment by wtgodbe** [comment]
> Think we can close this.

Yep - I think the Helix test failure is real, though, so I'm using the CI job to look into it. Will close this and replace with another PR to fix the test once I figure it out

**Comment by BrennanConroy** [comment]
A template test hang on Ubuntu. I tried to open it in VS (since that is supposedly supported now), but failed :(

**Comment by wtgodbe** [comment]
I saw the same thing in another PR, which is why I thought it was real, but maybe it's transient. I'll retry here, and kick the other servicing-approved 5.0 PRs just in case.

**Pull request closed without merging**